EventListener/ExceptionListener.php

```diff
@@ -34,15 +34,20 @@ class ExceptionListener implements EventSubscriberInterface
 {
     private $exceptionListener;
     private $dispatcher;
+    private $innerExceptionListener;
 
-    public function __construct($controller, ?LoggerInterface $logger, EventDispatcherInterface $dispatcher)
+    public function __construct($controller, ?LoggerInterface $logger, $debug = false, EventDispatcherInterface $dispatcher, $innerExceptionListener = null)
     {
-        if (class_exists(ErrorListener::class)) {
-            $this->exceptionListener = new ErrorListener($controller, $logger);
-        } else {
-            $this->exceptionListener = new LegacyExceptionListener($controller, $logger);
-        }
+        $this->exceptionListener = new ErrorListener($controller, $logger, $debug);
         $this->dispatcher = $dispatcher;
+        $this->innerExceptionListener = $innerExceptionListener;
+    }
+
+    public function logKernelException(ExceptionEvent $event)
+    {
+        if ($this->innerExceptionListener) {
```

Comment:
we should be able to log the exception even if the inner listener is null. 
I'd suggest call to `$this->exceptionListener->logKernelException()`. Same for `onControllerArguments()` method.

Reply:
the reason I didn't do this is because I wanted to keep this listener scoped with its current focus. if you want logging, then imho you should keep the core listener enabled. but I am open to reconsider.